pkg/transformer/openshift/openshift.go

```diff
@@ -200,6 +200,13 @@ func (o *OpenShift) initImageStream(name string, service kobject.ServiceConfig,
 // initBuildConfig initialize Openshifts BuildConfig Object
 func initBuildConfig(name string, service kobject.ServiceConfig, repo string, branch string) (*buildapi.BuildConfig, error) {
 	contextDir, err := getAbsBuildContext(service.Build)
+	envList := []kapi.EnvVar{}
+	for envName, envValue := range service.BuildArgs {
+		if *envValue == "\x00" {
+			*envValue = os.Getenv(envName)
+		}
+		envList = append(envList, kapi.EnvVar{Name: envName, Value: *envValue})
```

Comment:
adding api instead of kapi  duplicates other objects and distrubs everything, so I used kapi